charts/common/templates/_pod.tpl

```diff
@@ -1,3 +1,13 @@
+{{- define "accelleran.common.pod" -}}
+apiVersion: v1
+kind: Pod
+{{ toYaml (merge
+    (fromYaml (include "accelleran.common.metadata" .))
```

Comment:
Do we need to provide empty default values or fallbacks if the included templates for metadata or pod.tpl is missing?

Reply:
They are defined as part of this chart, so we can assume it will always return something as there are required fields to be returned. E.g. for `metadata` the name is required. If something would be missing, then the kubernetes api would be complaining as well and then also the tests will fail that use this etc.